frontend/online-shop/src/styles/ProductItemStyle.tsx

```diff
@@ -0,0 +1,48 @@
+import { makeStyles } from "@material-ui/core";
+
+export const useProductItemStyles = makeStyles({
```

Comment:
As far as I see you haven't used the custom Theme in any of your style files. You can pass it as a parameter within `createStyles` and then use its properties for example: `theme.palette.primary.main`